examples/graphql-scalar-id/graphql_id_scalar.bal

```diff
@@ -0,0 +1,19 @@
+import ballerina/graphql;
+
+service /graphql on new graphql:Listener(9090) {
+    private User[] users = [
+        {id: 1, name: "Walter White", age: 57},
+        {id: 2, name: "Jesse Pinkman", age: 28}
+    ];
+
+    isolated resource function get user(@graphql:ID int userId) returns User {
```

Comment:
Can't we just say `id` here? When filtering the `user` field, `id` implies it is in fact the `userId`.

```suggestion
    isolated resource function get user(@graphql:ID int id) returns User {
```

Reply:
Fixed in [73bf589](https://github.com/ballerina-platform/ballerina-distribution/pull/5188/commits/73bf589184b148f87e83d105386f7ec2710527b7)